sections/main-cart-items.liquid

```diff
@@ -46,13 +46,15 @@
                 <tr class="cart-item" id="CartItem-{{ item.index | plus: 1 }}">
                   <td class="cart-item__media">
                     {% if item.image %}
-                      <img class="cart-item__image"
-                        src="{{ item.image | img_url: '150x' }}"
-                        alt="{{ item.image.alt | escape }}"
-                        loading="lazy"
-                        width="75"
-                        height="{{ 75 | divided_by: item.image.aspect_ratio | ceil }}"
-                      >
+                      <a href="{{ item.product.url }}">
```

Comment:
Should this be `item.url` (same for product title) to allow the buyer to go directly to the same variant as they currently have in their cart?